### DISCUSSION TITLE
This Tutorial is very helpful...

### DISCUSSION THREAD

**Discussion by zawhtut** [microsoft/hack-together#89] [Show and tell]
Hi friends,

If you are still figuring out trying to access Graph API that requires tokens..., check out this tutorial...
[https://learn.microsoft.com/en-us/graph/tutorials/dotnet-app-only](https://learn.microsoft.com/en-us/graph/tutorials/dotnet-app-only)

Cheers...
😃

**Comment by aycabas** [comment]
Thanks for sharing @zawhtut 🙏
